sentry-android-core/src/main/java/io/sentry/android/core/DefaultAndroidEventProcessor.java

```diff
@@ -347,8 +350,10 @@ private void setArchitectures(final @NotNull Device device) {
     if (memInfo != null) {
       // in bytes
       device.setMemorySize(getMemorySize(memInfo));
-      device.setFreeMemory(memInfo.availMem);
-      device.setLowMemory(memInfo.lowMemory);
+      if (applyScopeData) {
```

Comment:
`m`: I think it would be nice to add a comment here or somewhere else why we do this. Ideally, we could extract the bottom line from https://github.com/getsentry/sentry-java/issues/1395, so we know in the future why we did this.

Reply:
makes sense, will do